lib/config/configuration_test.go

```diff
@@ -765,6 +794,30 @@ SREzU8onbBsjMg9QDiSf5oJLKvd/Ren+zGY7
 	require.Equal(t, 1, *cfg.Auth.KeyStore.SlotNumber)
 	require.Equal(t, "example_pin", cfg.Auth.KeyStore.Pin)
 	require.ElementsMatch(t, []string{"ca-pin-from-string", "ca-pin-from-file1", "ca-pin-from-file2"}, cfg.CAPins)
+
+	require.True(t, cfg.Databases.Enabled)
+	require.Empty(t, cmp.Diff(cfg.Databases.AzureMatchers,
+		[]services.AzureMatcher{
+			{
+				Subscriptions:  []string{"sub1", "sub2"},
+				ResourceGroups: []string{"group1", "group2"},
+				Types:          []string{"postgres", "mysql"},
+				Regions:        []string{"eastus", "centralus"},
+				Tags: map[string]apiutils.Strings{
+					"a": {"b"},
+				},
+			},
+			{
+				Subscriptions:  []string{"*"},
+				ResourceGroups: []string{"*"},
+				Types:          []string{"postgres", "mysql"},
+				Regions:        []string{"westus"},
+				Tags: map[string]apiutils.Strings{
+					"c": {"d"},
+				},
+			},
+		},
+		cmp.AllowUnexported(Service{})))
```

Comment:
The the `cmp.AllowUnexported(Service{})))` option is needed ? We are compering the AzureMatchers so  `AllowUnexported(Service{})` looks superfluous

Reply:
It is superfluous, I'm not sure how it slipped in there. Removing